mephisto/client/review/review_server.py

```diff
@@ -324,8 +324,9 @@ def all_task_data():
             data = {"data": current_data if not finished else None, "id": counter - 1}
             return jsonify({"data": data, "mode": MODE, "finished": finished})
 
-    @app.route("/")
-    def index():
+    @app.route("/", defaults={"id": 0})
```

Comment:
Good point. The id is set to 0 because the parameter itself is not used. It is just a placeholder for the URL structure of the route. Nevertheless it would make more sense that I make it None.